core/src/main/scala/coulomb/quantity.scala

```diff
@@ -59,7 +60,15 @@ object qopaque:
         abstract class Applier[U]:
             def apply[V](v: V): Quantity[V, U]
         object Applier:
-            given [U]: Applier[U] = new Applier[U] { def apply[V](v: V): Quantity[V, U] = v } 
+            given [U]: Applier[U] = new Applier[U] { def apply[V](v: V): Quantity[V, U] = v }
+
+        inline given [V, U](using ord: Order[V]): Order[Quantity[V, U]] = ord
+
+    private[qopaque] sealed class QuantityLowPriority0 extends QuantityLowPriority1:
+        inline given [V, U](using hash: Hash[V]): Hash[Quantity[V, U]] = hash
+
+    private[qopaque] sealed class QuantityLowPriority1:
+        inline given [V, U](using eq: Eq[V]): Eq[Quantity[V, U]] = eq
```

Comment:
We have to do this annoying prioritisation thing to avoid ambiguous implicits, since both `Order` and `Hash` extends `Eq`.